src/actions.ml

```diff
@@ -1795,12 +1795,23 @@ let update_pr ~bot_info (pr_info : issue_info pull_request_info) ~gitlab_mapping
           ~pr_number:pr_info.issue.number ~base:local_base_branch
           local_head_branch)
   >>= fun ok ->
+  let label = "needs: rebase" in
+  let issue = pr_info.issue.issue in
+  GitHub_queries.get_label ~bot_info ~owner:issue.owner ~repo:issue.repo ~label >>= fun label_id ->
+  GitHub_queries.get_label ~bot_info ~owner:issue.owner ~repo:issue.repo ~label:"stale" >>= fun stale_id ->
```

Comment:
Important: make it robust to these labels not existing. With the REST API call, this was fine. But now, the queries for getting the labels IDs need to succeed both to check whether one of these labels exists. At the current time (and this will probably continue being the case in the future), most repositories relying on coqbot do not have the stale label (they all have the `needs: rebase` label because the REST API call did also create the label if it didn't already exist, so this one change will need to be documented).